src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -170,7 +171,10 @@ private void ExpandCategory(TreeViewItem sender, NodeCategoryViewModel selectedC
             // If categoryToBeExpanded is null, that means not category button, but class button was clicked.
             // During loop we will find out to which category this clicked class belongs.
             if (categoryToBeExpanded != null)
+            {
                 categoryToBeExpanded.IsExpanded = !categoryToBeExpanded.IsExpanded;
+                hasSelectedClassBeenFound = true;
+            }
```

Comment:
I have the same question as I had previously, instead of

```csharp
categoryToBeExpanded.IsExpanded = !categoryToBeExpanded.IsExpanded;
```

Why don't we do the following since it is always going to be expanded?

```csharp
categoryToBeExpanded.IsExpanded = true;
```